main.go

```diff
@@ -79,7 +79,14 @@ func main() {
 
 	// Run http server concurrently
 	// Load routes for the server
-	var mux http.Handler = NewMux(db)
+	var mux http.Handler
+
+	// Allow for CSRF to "pass" even over non-https. Hence, unsafe.
+	if noHTTPS {
+		mux = NewMux(true, db)
+	} else {
+		mux = NewMux(false, db)
+	}
```

Comment:
This could probably just be `NewMux(noHTTPS, db)` unless we expect other things to influence whether its unsafe or not in the future.

Reply:
Very true